testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/WithPluginValidation.groovy

```diff
@@ -102,7 +102,7 @@ trait WithPluginValidation {
                     !(it.outcome in [
                         TaskOutcome.NO_SOURCE,
                         TaskOutcome.SKIPPED
-                    ]) && it.path.contains(taskPattern)
```

Comment:
💭 This bit is annoying, but otherwise Gradle smoke tests kept finding a `:plugins` project in the old version of Gradle.  I'll submit a followup PR to remove this after a new old version of Gradle without `plugins` is available.